egs/librispeech/ASR/pruned_transducer_stateless2/beam_search.py

```diff
@@ -1219,8 +1219,10 @@ def fast_beam_search_with_nbest_rescoring(
     temperature: float = 1.0,
 ) -> Dict[str, List[List[int]]]:
     """It limits the maximum number of symbols per frame to 1.
-    A lattice is first obtained using modified beam search, and then
-    the shortest path within the lattice is used as the final output.
+    A lattice is first obtained using fast beam search, num_path are selected
+    and rescored using a given language model. The shortest path within the
```

Comment:
Can you help update line 1230:

HLG -> LG

There is no H here.

There may be HLG in other places in this file. Please also replace them with LG. Thanks.

Reply:
Done !